packages/contentful--react-apps-toolkit/src/useAutoResizer.tsx

```diff
@@ -0,0 +1,17 @@
+import { FieldExtensionSDK } from '@contentful/app-sdk';
+import { useEffect } from 'react';
+import { useSDK } from './useSDK';
+
+/**
+ * Invokes the FieldExtensionSDK autoResizer.
+ * Must be wrapped by SDKProvider.
+ * Can only be used when app is rendered in field, sidebar or entry editor location.
+ */
+export function useAutoResizer(): void {
+  const sdk = useSDK<FieldExtensionSDK>();
```

Comment:
To be more precise about the types here.

```
  const sdk = useSDK<SidebarExtensionSDK | FieldExtensionSDK | DialogExtensionSDK>();
```

Reply:
Yep perfect, added this in 👍